src/xcode/ENA/ENA/Source/Services/PPAccessControl/PPACService.swift

```diff
@@ -103,6 +161,16 @@ class PPACService: PrivacyPreservingAccessControl {
 		return storedToken
 	}
 
+	private var apiTokenSRS: TimestampedToken {
+		guard let storedToken = store.ppacApiTokenSrs
+		else {
+			let newToken = generateAndStoreFreshAPIToken()
+			store.ppacApiTokenEls = newToken
```

Comment:
⚠️ Should be `store.ppacApiTokenSrs`

Reply:
Fixed with https://github.com/corona-warn-app/cwa-app-ios/pull/4900